CHANGELOG.md

```diff
@@ -1,56 +1,83 @@
-##2015-09-02 - Unsupported Release 0.8.1
+## 2016-01-14 - Supported Release 1.0.0
+### Summary
 
-###Summary
+First supported release! Performance is 2x+ by reusing PowerShell instance! Support PSCredentials, KeyValuePair and other Embedded Instance Classes. Update DSC Modules to latest available. Notify on Reboot required (and handle it with known named Puppet reboot resource).
+
+### Features
+
+- Reuse PowerShell instance for increased performance ([MODULES-2709](https://tickets.puppetlabs.com/browse/MODULES-2709))
+- Support PSCredentials, KeyValuePair and other Embedded Instance Classes ([MODULES-2178](https://tickets.puppetlabs.com/browse/MODULES-2178))
+- Remove LCM Refresh Mode Disabled Requirement ([MODULES-2575](https://tickets.puppetlabs.com/browse/MODULES-2575))
+- Include Supportable DSC Resource Types ([MODULES-2244](https://tickets.puppetlabs.com/browse/MODULES-2244))
+- Update DSC Modules to latest available (84a467c) and reduce the size by removing unnecessary files ([MODULES-2692](https://tickets.puppetlabs.com/browse/MODULES-2692)) ([MODULES-2777](https://tickets.puppetlabs.com/browse/MODULES-2777))
```

Comment:
Could we link to the commit we did it? IIRC we put the commit sha link in the commit msg so they can link directly to where/when we got it from.